packages/generate/src/queries.ts

```diff
@@ -38,6 +45,21 @@ currently supported.`);
 
   console.log(`Detected schema directory: ${params.schemaDir}`);
   const matches = await getMatches(root, params.schemaDir);
+
+  const errs:string[] = [];
+  function printSummary(){
+    const passedCount = matches.length - errs.length
+    const summary = `\n\x1b[${errs.length > 0 ? '1;31m' : '1;32m'}${passedCount}/${matches.length} queries passed${errs.length ? ', '+errs.length+' failed'  : '!'}`
+    console.log(summary)
+    for (const err of errs) {
+        console.log('\x1b[22m')
+        console.error(err)
+    }
+    if(errs.length > 1) console.log(summary)
```

Comment:
Did you mean to log the summary twice? Seems like unconditionally logging here at the end is just as good?

Reply:
I see your update, but what I had in mind was always logging the summary at the end and never logging it at the beginning.

So:

```ts
console.log(summary);
```

In other words, what I had in mind for the final output here is:

1. Printing each file as their being evaluated, capturing any errors.
2. At the end, printing all of the errors.
3. Print the summary.

What do you think?